README.rst

```diff
@@ -76,13 +76,18 @@ In addition to the pure API implementation, this library features a number of hi
 make the development of bots easy and straightforward. These classes are contained in the
```

Comment:
maybe we should also mention all our shortcuts here? e.g.

_In addition to the pure API implementation, this library features convenient shortcuts to save you from re-passing variables and_

this comment should go a line higher, github doesnt let me.

Reply:
nice idea. Tried to tie it in in the same sentence :)